pkg/kubelet/apis/kubeletconfig/v1beta1/defaults.go

```diff
@@ -63,118 +82,95 @@ func SetDefaults_KubeletConfiguration(obj *KubeletConfiguration) {
 	if obj.Authorization.Webhook.CacheUnauthorizedTTL == zeroDuration {
```

Comment:
(above, [line 77](https://github.com/kubernetes/kubernetes/pull/53833/files#diff-128d1b6a626039bd139783f5354e8cb2R77)): `obj.Authorization.Mode = KubeletAuthorizationModeAlwaysAllow`

I would like to change this someday.

Reply:
https://github.com/kubernetes/kubernetes/pull/59666